problems/median.rb

```diff
@@ -1,4 +1,4 @@
-# Write a method that returns the median of elements in an array
+# Write an `Array#median` method that returns the median element in an array
```

Comment:
Move this prompt into the class statement, this is how it's done on other problems and the real assessment and we should be consistent. ex.
```
class Array
     # prompt here
     def method_name
     end
end
```